docs/docs/tutorials/creating-new-environments.md

```diff
@@ -1,181 +1,94 @@
+To navigate to conda-store:
 
-4. Click on the card for your newly created environment. The UI will display the environment specification:
+* From Nebari Home, click on **"Environment Management"** under "Service"
+* From JupyterLab, click on `Nebari` in the menu bar and go to **"Environments"**
+* From anywhere, go to URL: `https://<your-nebari-domain>/conda-store`
 
-   ![conda-store - Environment details page](/img/tutorials/conda_store_env_details.png)
+If not logged in to conda-store, click on the **"Log in"** button in th left sidebar and authenticate similar to the [Nebari login][login-keycloak]. This is required to be able to access many `conda-store` features.
 
-   If this is the first time visiting this page, the environment may still be in the process of building.
-   The page will automatically update when the build is complete.
+## Create, edit, and manage environments
 
-   From this page, you can `Edit` your YAML specification, or even `Delete` the environment.
+Go through the following conda-store (UI) tutorials
+on using the graphical interface for various actions:
 
-   :::note
-   `conda-store` tracks all the environments behind the scenes for the users. Even if a user "deletes" an environment,
-   it will still exist in the store. This ensures admins always have access to environment history.
-   :::
+* [Create new environments][cs-create-env]
+* [Edit & delete existing environments][cs-edit-delete-env]
+* [Switch environment versions][version-control]
 
-5. Now let's take a closer look at the `conda` build detail, to do this, click on the build number link in the `conda-store` interface.
-   This will display a new page with the metadata about the environment including the time, size and status.
-   Also included is your original YAML specification and a list of all the installed dependencies, their version and the used `conda` channels.
+## Default namespaces in Nebari
 
-   ![conda-store UI - Sample environment build details page: showing build details, specification file, and conda packages](/img/tutorials/conda_store_build_details_top.png)
+A default Nebari deployment/instance has the following namespaces corresponding to [Nebari groups][configure-keycloak-groups]:
 
-   Scroll down to the bottom of this list, and you'll see a section called `Conda Environment Artifacts`.
-   This is where you can download your own copy of the YAML file, a [conda-lock](https://conda.github.io/conda-lock/) file, and a `tar.gz` archive of the environment.
+<!-- Verify the roles and actions -->
 
-   ![conda-store UI - Sample environment build details page: showing the conda environment artifacts](/img/tutorials/conda_store_build_details_bottom.png)
+* `analyst` namespace - Users in the `analyst` group can view and `admin` group can view+edit the environments in this namespace
+* `developer` namespace - Users in the `developer` and `admin` groups can view+edit the environments in this namespace
```

Comment:
I'm pretty sure developer is required for access to the dask workers.

Reply:
I was about to suggest some changes, but noticed this will be confusing with the keycloak roles and the concepts of namespaces intertwined, so I will think about how to rewrite this a bit